include/Infectorpp/InfectorContainer.hpp

```diff
@@ -1,188 +1,178 @@
-    void rollback_multiple_inheritance_inner();
-    template <typename Contract, typename Next, typename... Others>
-    void rollback_multiple_inheritance_inner();
-    template <typename T, typename Contract, typename... Others>
-    void rollback_multiple_inheritance();
-
-    template <typename T>  void must_have();
-    template <typename T>  void must_not_have();
-    template <typename T>  void launch_exception();
-    template <typename T>  std::shared_ptr<T> buildSingle_delegate();
-    template <typename T>  std::unique_ptr<T> build_delegate();
-
-    template<typename Dummy>
-    bool tryToGetSize( RecursionLimit *, int *);
-    template<typename Actual, typename Next, typename...Others>
-    bool tryToGetSize( RecursionLimit *, int *);
-
-    template<typename Dummy=DummyClass> // fix for bug reported by Michael Bischof
-    void setKnownSizeForType( std::type_index t, bool known, int size);
-    template<typename Dummy=DummyClass> // fix for bug reported by Michael Bischof
-    void processRecursionWeb();
-
-    template <typename OBJ> friend class UniqueOrShared;
-    struct EmplaceContext;
-    /** Too long explain why this is so usefull.*/
-    template <typename OBJ>
-    class UniqueOrShared{
-        Container * ioc=nullptr;
-        EmplaceContext * context;
-    public:
-        UniqueOrShared(Container * ptr, EmplaceContext * c)
-        :ioc(ptr)
-        ,context(c){}
-
-        operator std::shared_ptr<OBJ>(){
-            return std::move(std::shared_ptr<OBJ>(std::move(ioc->buildSingle_delegate<OBJ>())));
-        }
-
-        operator std::unique_ptr<OBJ>(){
-            return std::move(std::unique_ptr<OBJ>(std::move(ioc->build_delegate<OBJ>())));
-        }
-    };
-
-    struct Binding{
-        std::type_index type;
-        bool            single;
-        Binding():type( typeid(Binding)){}
-        Binding( std::type_index a, bool b): type(a),single(b){}
-        Binding( const Binding & other) = default;
-    };
-
-    struct EmplaceContext{
-        bool factory;
-    };
-
-    std::unordered_map< std::type_index, IAnyShared*>       singleIstances;
-    std::unordered_map< std::type_index, Binding>           typeMap;
-    std::unordered_map< std::type_index, std::function<void*(EmplaceContext*)>>
-                                                            callbacks;
-
-    std::unordered_map< std::type_index, int>               typeInfoMap;
-    std::unordered_map< std::type_index,
-                        std::function< bool( RecursionLimit *, int *)>>
-                                                            recursionMap;
-    std::list< std::function< bool( RecursionLimit *, int *)>>
-                                                            recursionWaitList;
-
-    //RecursionLimit limit;
-};
-} // namespace Infector
-
-#include "Infector_details.hpp"
-#include "Infector_private.hpp"
-#include "Infector_multi.hpp"
-#include "Infector_single.hpp"
+/* Copyright (c) 2013 Dario Oliveri
```

Comment:
Are there any functional changes in this file? It seems like line-endings for this file was changed from Unix-style to Windows-style. Can you please confirm and revert changes to these files?